messaging_sample/src/main/java/com/zendesk/messaging_sample/Global.java

```diff
@@ -62,10 +63,9 @@ public void onCreate() {
         );
 
         // b). JWT (Must be initialized with your JWT identifier)
-        Zendesk.INSTANCE.setIdentity(new JwtIdentity("{JWT User Identifier}"));
-
+        //Zendesk.INSTANCE.setIdentity(new JwtIdentity("{JWT User Identifier}"));
```

Comment:
is this intended?

Reply:
Yes, in the past some customer were confused why some app would crash as the identity becomes JWT if we leave this in, and most of them don't test with it setup. Just to error-proof the app more, I think it's clear enough to have the "basic" option there and in comment the more advanced one